lib/email_inquire/validator/commonly_mistaken_domain.rb

```diff
@@ -9,6 +9,35 @@ class CommonlyMistakenDomain < Base
       MISTAKES = {
         /google(?!mail)/ => "gmail.com",
         /windows.*\.com/ => "live.com",
+        /hormail.com/ => "hotmail.com",
```

Comment:
take care of the point, I mean scape the dot because if not is can be "anything" com